cmd/backup/config.go

```diff
@@ -148,3 +149,21 @@ func (n *NaturalNumber) Decode(v string) error {
 func (n *NaturalNumber) Int() int {
 	return int(*n)
 }
+
+type NaturalNumberZero int
```

Comment:
```suggestion
type WholeNumber int
```
Maybe that's a bit less clunky?